docs/lab/01.md

```diff
@@ -2,4 +2,240 @@
+
+$$
+U = \frac{L}{Q};
+$$
+
+U = Electromotive voltage;  
+L = Mechanical work of electric force;  
+Q = Electric charge; 
+
+The unit of measurement of electric voltage in the SI[^si] is the **Volt**(V)
+
+$$
+[U]_{SI} = V(Volt) \\
+[L]_{SI} = J(Joule) \\
+[Q]_{SI} = C(Coulomb) \\
+$$
+
+:::info
+
+Electronic voltage is always measured between two points in a circuit. In general, voltages are measured against a reference point called circuit mass **(GND)**. Circuit mass is a convention and represents the point which potential is consider null. 
+
+:::
+
+#### Electric resistance
+
+Electrical resistance is a physical quantity that expresses the property of a material to oppose the passage of electric current. The SI[^si] unit of resistance is the **Ohm**, noted &ohm;.
+
+$$
+[R]_{SI} = \Omega(Ohm)
+$$
+
+#### Intensity of electric current
+
+The intensity of the electric current, also called the electric current, is a scalar physical quantity equal to the variation of the electric charge that crosses the section of a conductor in the unit of time. The unit of measurement in SI[^si] is the **Ampere**(A).
+
+$$
+[I]_{SI} = A(Ampere)
+$$
+
+### Ohm's law
+
+The intensity(I) of the electric current passing through a resistor is directly proportional to the voltage(U) applied to the resistor and inversely proportional to its resistance value(R).
+
+<div align="center">
+![Ohm's Law](images/ohm.png)
+</div>
+
+$$
+I = \frac{U}{R};
+$$
+
+I = intensity of electric current(A)   
+U = electrical voltage(V)   
+R = circuit's resistance(&ohm;)
+
+### Kirchhoff's First Law
+
+The sum of currents flowing into a junction is always equal to the sum of currents flowing out of the same junction.
+
+$$
+\sum_{k=1}^{n} I_k = 0
+$$
+
+<div align="center">
+![Kirchhoff's First Law](images/kirchhoff_1.png)
+</div>
+
+$$
+I_1 + I_2 = I_3 + I_4 + I_5 \\
+I_1 + I_2 - I_3 - I_4 - I_5 = 0
+$$
+
+### Kirchhoff's Second Law
+
+Around any closed loop in a circuit, the directed sum of potential differences across components is zero.
+
+$$
+\sum_{k=1}^{n} U_k = 0
+$$
+
+<div align="center">
+![Kirchhoff's Second Law](images/kirchhoff_2.png)
+</div>
+
+$$
+-U_A - U_B - U_C + U_D + U_E = 0
+$$
+
+### Voltage divider
+
+There are many types of voltage dividers, being named according to the type of component elements: resistive divider, capacitive divider, compensated divider, etc...   
+The **resistive voltage divider** is obtained by applying a voltage `E` to a group of resistors in series and thus a fraction of the voltage applied to one of the resistors of the group can be obtained.
+
+<div align="center">
+![Voltage divider on GND](images/voltage_divider_gnd.png)
+</div>
+
+$$
+V_{out} = V_{in} * \frac{R_{2}}{R_{1} + R_{2}};
+$$
+
+:::warning
+Voltage divider can be considered a power source only if it operates in the empty state. It is **not recommended** to use a voltage divider to supply a circuit. It can be used to provide reference points.
```

Comment:
I would explain why => the internal resistance is high and energy is lost due to heating.